stdlib/src/lib.rs

```diff
@@ -99,6 +103,29 @@ impl Default for StdLibrary {
     }
 }
 
+/// Signs the provided message with the supplied secret key and returns the encoded calldata
+/// expected by `stdlib::crypto::dsa::ecdsa::verify_bis`.
+///
+/// The returned felt sequence mirrors the interface of `falcon_sign`:
+///
+/// 1. The compressed secp256k1 public key encoded as 9 packed-u32 felts (33 bytes total).
+/// 2. The ECDSA signature encoded as 17 packed-u32 felts (66 bytes total).
+///
+/// The two chunks are concatenated as `[PK_BYTES || SIG_BYTES]` so they can be streamed straight to
+/// the advice provider before invoking `verify_bis`.
+pub fn ecdsa_sign(sk: &mut SecretKey, msg: Word) -> Vec<Felt> {
```

Comment:
Does `SecreteKey` need to be mutable here? I think if you run `cargo update` we should get the version of `miden-crypto` that doesn't require this.

Also, we may not always have access to the private key - so, in addition to this function, it would be great to expose a function that converts a public key and a signature into a vector of bytes. Or, even better, if we could convert message and signature into a vector of bytes (maybe using EC recover?) - maybe something like:

```rust
pub fn prepare_ecdsa_signature(pk: &PublicKey, msg: Word) -> Vec<Felt> {
    ...
}
```

For v0.20 release, it would be great to find a better place for these (and for `falcon_sign()`) functions. Exporting them from the library root feels a bit weird.